compute/instances/troubleshooting/fix-unreachable-ipv6-rhel-based-instance.mdx

```diff
@@ -0,0 +1,92 @@
+---
+meta:
+  title: Fix unreachable IPv6 RHEL based Instance after transition to routed ip
+  description: This page helps recover or avoid unreachable IPv6 Instance after transition to routed ip
+content:
+  h1: Fix unreachable IPv6 RHEL based Instance after transition to routed ip
+  paragraph: This page helps recover or avoid unreachable IPv6 Instance after transition to routed ip
+tags: centos-stream rockylinux almalinux network-manager ipv6 routed ip
+dates:
+  validation: 2023-11-16
+  posted: 2023-11-16
+categories:
+  - compute
+---
+
+RHEL based Instances created before November 15th, 2023 require a NetworkManager configuration that enable correct IPv6 configuration after a transition to routed IP is performed. If your Instance is configured only with an IPv6 address, it will become unreachable after the migration. It is important to add the NetworkManager configuration so IPv6 configuration remains correct.<br />
```

Comment:
```suggestion
RHEL-based Instances created before November 15th 2023 require a NetworkManager configuration that enables correct IPv6 configuration, after a transition to routed IP is performed. If your Instance is configured only with an IPv6 address, it will become unreachable after the migration. It is important to add the NetworkManager configuration so IPv6 configuration remains correct.<br />
```